ui/src/shared/copy/notifications.ts

```diff
@@ -948,3 +948,42 @@ export const deleteBucketFailed = (): Notification => ({
   ...defaultErrorNotification,
   message: 'Failed to delete bucket',
 })
+
+// Limits
+export const writeLimitReached = (errorMessage: string): Notification => ({
+  ...defaultErrorNotification,
+  message: `You have reached a write limit on this resource: ${errorMessage}.`,
+  duration: INFINITE,
+})
```

Comment:
I wonder if `writeLimitReached` and `readLimitReached` should become convenience mehtods for `readWriteLimitReached`?

Reply:
I was using the writeLimitReached for line protocol writes, so it felt reasonable here. I'm expecting feedback on error messages.

Reply:
+1

Reply:
I really like the message you've written on L974. Seems like a good wording until we introduce response headers that will return more specific info.